README.md

```diff
@@ -33,45 +33,75 @@ sample rokudeploy.json
     "password": "securePassword"
 }
 ```
-## Usage
+## CLI Usage
 
-From a node script
-```javascript
-var rokuDeploy = require('roku-deploy');
+### Deploy a zip package
+Deploy a .zip package of your project to a roku device
+```shell
+npx roku-deploy deploy --host ${ip-of-roku} --password ${password} --rootDir '.' --outDir './out'
+```
 
-//deploy a .zip package of your project to a roku device
-rokuDeploy.deploy({
-    host: 'ip-of-roku',
-    password: 'password for roku dev admin portal'
-    //other options if necessary
-}).then(function(){
-    //it worked
-}, function(error) {
-    //it failed
-    console.error(error);
-});
+
+### Create a signed package of your project
+```shell
+npx roku-deploy deploy package --host ${ip-of-roku} --password ${password} --signingPassword ${signing-password}
 ```
-Or
-```javascript
-//create a signed package of your project
-rokuDeploy.deployAndSignPackage({
-    host: 'ip-of-roku',
-    password: 'password for roku dev admin portal',
-    signingPassword: 'signing password'
-    //other options if necessary
-}).then(function(pathToSignedPackage){
-    console.log('Signed package created at ', pathToSignedPackage);
-}, function(error) {
-    //it failed
-    console.error(error);
-});
+
+### Other uses
+Stage the root directory
+```shell
+npx roku-deploy stage --stagingDir ${stagingDir} --rootDir ${rootDir}
+```
+Zip the contents of a given directory
+```shell
+npx roku-deploy zip --stagingDir ${rootDir} --outDir ${outDir}
+```
+Press the Home key
+```shell
+npx roku-deploy keyPress --key 'Home' --host ${ip-of-roku} --remotePort ${remotePort} --timeout ${timeout}
+```
+Sideload a build
+```shell
+npx roku-deploy sideload --host ${ip-of-roku} --password ${password} --outDir ${outDir} 
+```
+
+Convert to SquashFS
+```shell
+npx roku-deploy squash --host ${ip-of-roku} --password ${password}
+```
+Create a signed package
+```shell
+npx roku-deploy sign --host ${ip-of-roku} --password ${password}
 ```
 
+Here are all the possible cli commands:
+- bundle
+- deploy
+- package
+- exec
+- keypress
+- keyup
+- keydown
+- sendText
+- stage
+- sideload
+- squash
+- rekey
+- createSignedPackage
+- deleteDevChannel
+- screenshot
+- getDeviceInfo
+- getDevId
+- zip
+
+
+## Node script Usage
```

Comment:
Let's include javascript examples for each cli command shown above. (seems like there are a few less in the javascript examples)

Reply:
addressed